.changeset/lemon-seahorses-cover.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-react": minor
+---
+
+All instances of `<SvgStatusError>` have been updated to reflect the new octagon shape from the latest version of iTwinUI-icons. Some components affected: `Alert`, `Toast`, `StatusMessage`.
```

Comment:
Is `<SvgStatusError>` an itwinui-react package implementation detail? If so, maybe the changeset can use some other name to refer to the error icon.

Reply:
Technically yes, but it's publicly exposed in `@itwin/itwinui-icons-react` with the same name, so this is probably the best way to refer to it.